### PR TITLE
docs(metrics): add additional serviceMonitor helm variables

### PR DESCRIPTION
**Pull request by ejweber**
#### Which issue(s) this PR fixes:

longhorn/longhorn#8142

#### Additional documentation or context

~Will hold off on this one until https://github.com/longhorn/longhorn/pull/8985 and its backports merge so I can make all website changes in one go.~

### CODE REVIEW

File: content/docs/1.6.3/references/helm-values.md [hunk 1]
```diff
@@ -160,6 +160,18 @@ You can install Longhorn in an air-gapped environment with a private registry. F
 | privateRegistry.registryUrl | URL of a private registry. When unspecified, Longhorn uses the default system registry. |
 | privateRegistry.registryUser | User account used for authenticating with a private registry. |
 
+### Metrics Settings
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| metrics.serviceMonitor.additionalLabels | object | `{}` | Setting that adds additional labels to the Prometheus ServiceMonitor. |
+| metrics.serviceMonitor.annotations | object | `{}` | Setting that adds annotations to the Prometheus ServiceMonitor. |
```
Comment (jillian-maroket) [inline content/docs/1.6.3/references/helm-values.md:168]:
```suggestion
| metrics.serviceMonitor.additionalLabels | `{}` | Additional labels for the Prometheus ServiceMonitor resource. |
| metrics.serviceMonitor.annotations | `{}` | Annotations for the Prometheus ServiceMonitor resource. |
```

Comment (jillian-maroket) [reply content/docs/1.6.3/references/helm-values.md:168]: For brevity, we directly describe the main object and include "setting" only when the supported values are true/false.

File: content/docs/1.6.3/references/helm-values.md [hunk 2]
```diff
@@ -160,6 +160,18 @@ You can install Longhorn in an air-gapped environment with a private registry. F
...
+| metrics.serviceMonitor.enabled | bool | `false` | Setting that allows the creation of a Prometheus ServiceMonitor resource for Longhorn Manager components. |
+| metrics.serviceMonitor.interval | string | `""` | Interval at which Prometheus scrapes the metrics from the target. |
+| metrics.serviceMonitor.metricRelabelings | list | `[]` | Configures the relabeling rules to apply to the samples before ingestion. See Prometheus operator documentation for format. |
+| metrics.serviceMonitor.relabelings | list | `[]` | Configures the relabeling rules to apply the target’s metadata labels. See Prometheus operator documentation for format. |
```
Comment (jillian-maroket) [inline content/docs/1.6.3/references/helm-values.md:172]:
```suggestion
| metrics.serviceMonitor.metricRelabelings | list | `[]` | Configures the relabeling rules to apply to the samples before ingestion. See the [Prometheus Operator documentation](https://prometheus-operator.dev/docs/getting-started/introduction/) for formatting details. |
| metrics.serviceMonitor.relabelings | list | `[]` | Configures the relabeling rules to apply the target’s metadata labels. See the [Prometheus Operator documentation](https://prometheus-operator.dev/docs/getting-started/introduction/) for formatting details. |
```

Comment (jillian-maroket) [reply content/docs/1.6.3/references/helm-values.md:172]: I linked to the intro because I couldn't find the page with formatting info.

**Review by jillian-maroket — COMMENTED** [review]
Just a few nits. I did not touch the descriptions for `interval`, `metricRelabelings`, `relabelings`, and `scrapeTimeout` because you said they were [lifted directly](https://github.com/longhorn/longhorn/pull/8120#issuecomment-2223843200) from the Prometheus docs.

File: content/docs/1.6.3/references/helm-values.md (hunk 1 above)
Comment (ejweber) [reply content/docs/1.6.3/references/helm-values.md:168]: Note that we cannot remove the `object` type from the table, as it is generated by helm-doc and is present in all tables on this page.

Comment (jillian-maroket) [reply content/docs/1.6.3/references/helm-values.md:168]: Apologies, that was a mistake on my end. I don't know why those cells were removed.

File: content/docs/1.6.3/references/helm-values.md (hunk 2 above)
Comment (ejweber) [reply content/docs/1.6.3/references/helm-values.md:172]: Added a more direct link.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**